classes/options-general.php

```diff
@@ -858,31 +858,31 @@ class="radio-thumbnail<?php if ( 'TERRAIN' == $map_type ) {
 													<tr valign="middle">
 														<td>
 															<label
-																for="display_search"><?php _e( 'Display Search Form', 'simplemap' ); ?></label>
+																for="display_search"><?php _e( 'Display Search Form', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 143 characters

<!-- Comment by [SideCI](https://sideci.com) -->